suite-native/app/README.md

```diff
@@ -14,8 +14,11 @@ It's good to have some tools installed before you begin:
 ## Running app on Android
 
 1. Connect device or run emulator. For a physical device, it's recommended to use [adb over wifi](https://developer.android.com/studio/command-line/adb#connect-to-a-device-over-wi-fi-android-11+) because you will have free up a USB port to connect Trezor device.
-1. Run packager - `yarn start`
 1. Run native build - `yarn android`
+1. Run packager - `yarn start`
+
+### Optional:
+If you're using [Trezor User Env](https://github.com/trezor/trezor-user-env) while running the app, you might need to map the ports for data transport between Trezor emulator and Android device emulator by running `adb reverse tcp:21325 tcp:21325`
```

Comment:
Please add also `adb reverse tcp:8081 tcp:8081` which is needed to run app in emulator or over wifi adb

Reply:
https://github.com/trezor/trezor-suite/pull/10489/commits/03316f262b9ce73f01024fd708794da2d8e8b02d